### ISSUE TITLE
Make argument position irrelevant

### ISSUE THREAD

**Issue by norling** [NBISweden/sda-cli#199]

**Comment by pahatz** [comment]
If I'm not mistaken, it's the opposite. This works:
`./sda-cli download url_list.txt -c testing/s3cmd.conf -outdir bla `
while this doesn't: 
`./sda-cli download -c testing/s3cmd.conf -outdir bla url_list.txt`

**Comment by pahatz** [comment]
Yes, that would also be a good solution. Honestly, I don't see this as a big issue since the documentation is clear on the order they should come. In the PR's state right now, it should just be able to handle the wrong in the rest of the modules.
